src/main/java/frc/robot/utilities/pid/TrigonPIDFController.java

```diff
@@ -0,0 +1,85 @@
+package frc.robot.utilities.pid;
+
+import edu.wpi.first.wpilibj.controller.PIDController;
+
+public class TrigonPIDFController extends PIDController implements PIDFConfigurable {
```

Comment:
why not extend `TrigonPIDController`?

Reply:
1. TrigonPIDController has a local variable of pidCoefs and not PIDFCoefs
2. the main reason - there is simply nothing other than a few tuning functions to inherate and inherating leaves a ton of dimond of death issues

Reply:
Well, maybe there shouldn't be a PIDFCoefs class like I suggested before. A class using PID + feedforward should just have these two separate components.